### ISSUE TITLE
How to download translations with make.yml?

### ISSUE THREAD

**Issue by Plazik** [drush-ops/drush#1587]
I'm trying to download translation using Drush 7.0.0 and simple make file:

```
core: "7.x"
api: 2

projects:
  drupal:
    version: ~


translations:
  - ru
```

But I got an error:

```
drush make small.make.yml
Make new site in the current directory? (y/n): y
Beginning to build small.make.yml.               
drupal-7.39 downloaded.                          
Could not retrieve l10n update url for .  
```

With --debug:

```
drupal-7.39 downloaded. [3.78 sec, 7.12 MB]                                                            
Executing: wget -q --timeout=30 -O /tmp/download_fileQL3SG3 https://localize.drupal.org/l10n_server.xml
Undefined index: https://localize.drupal.org/l10n_server.xml make.project.inc:438 [5.5 sec, 7.03 MB]   
Undefined variable: project make.project.inc:442 [5.5 sec, 7.03 MB]                                    
Could not retrieve l10n update url for . [5.5 sec, 7.03 MB]          
```

**Comment by martinwittmann** [comment]
I just ran into the same issue. Looking a bit into it drush uses https://localize.drupal.org/l10n_server.xml to look for localization files, which returns a http 404.

You can fix it by changing the url in drush (/path/to/drush/commands/make/make.drush.inc:12) to http://ftp.drupal.org/files/translations/l10n_server.xml
See https://www.drupal.org/node/2484343 for more details/discussion...

**Comment by Plazik** [comment]
I think it was fixed in 86b4641b7f167612bb1a218760d3f9903dc22a1a.

**Comment by ngstigator** [comment]
Ubuntu 14.04.5 LTS ships with drush 5.10.0 which is fixable by https://github.com/drush-ops/drush/issues/1587#issuecomment-142677012 or drush upgrade.
